### PR TITLE
Semantic Search Support

### PR DESCRIPTION
**Pull request by farzad528**
Added Semantic Seach Support to the README

### CODE REVIEW

**Comment by farzad528** [comment]
@HeidiSteen @mattmsft plz approve.

File: multiple-search-services/README.md [hunk 1]
```diff
@@ -53,6 +53,12 @@ Open the **appsettings.json** file in your local copy of the sample application
 
     + Find the URI in the [search service's Overview portal page](https://docs.microsoft.com/azure/search/search-manage#overview-home-page).
 
+1. "semanticConfigurationName": "Semantic Configuration Name on your index":
+
+    + Required if you want to request a Semantic query. Find the Semantic Config in the [search service's desired index in the search explorer under the Configurations tab](https://learn.microsoft.com/en-us/azure/search/semantic-how-to-query-request?tabs=portal%2Cportal-query).
```
Comment (HeidiSteen) [inline multiple-search-services/README.md:58]: line 58 -- how about "Required if you want to request semantic reranking. You can choose an existing configuration or [create a new one in the portal](https://learn.microsoft.com/azure/search/semantic-how-to-query-request?tabs=portal%2Cportal-query)".  
Note the casing.
In the link, please remove "en-us" (it hard codes the english version of the article).

File: multiple-search-services/README.md [hunk 2]
```diff
@@ -53,6 +53,12 @@ Open the **appsettings.json** file in your local copy of the sample application
...
+
+    **Note**: You must create a Semantic Configuration first in the Portal or via the REST API before issuing a Semantic query. Additionally, you can find the list of query languages in the [List of supported languages](https://learn.microsoft.com/en-us/rest/api/searchservice/preview-api/search-documents#queryLanguage)
```
Comment (HeidiSteen) [inline multiple-search-services/README.md:60]: line 60 - if you accept the above suggestion, you can scope this note to just the query language consideration. "Semantic ranking comes with language requirements. See [List of supported language](https://learn.microsoft.com/rest/api/searchservice/preview-api/search-documents#queryLanguage) for details.

**Review by HeidiSteen — CHANGES_REQUESTED** [review]

File: multiple-search-services/appsettings.json [hunk 3]
```diff
@@ -1,16 +1,16 @@
 {
 
   "service 1": {
-    "adminKey": "Admin key for Search Service 1",
-    "searchEndpoint": "https://[search-service-1-name].search.windows.net",
-    "indexName": "good-books",
-    "semanticConfigurationName":  "name of semantic configuration, only required for semantic queries"
+    "adminKey": "WHvCso8u5yiHoVp9TnHU1vvus6MSeqaT0dSFHRKaErAzSeAKxYfY",
```
Comment (mattmsft) [inline multiple-search-services/appsettings.json:4]: Rotate your keys immediately

File: multiple-search-services/part1.json [hunk 4]
```diff
@@ -0,0 +1,188 @@
+{
+    "@odata.context": "https://fsunavala-ss-sandbox.search.windows.net/$metadata#indexes/$entity",
+    "@odata.etag": "\"0x8DAC331A066A901\"",
+    "name": "people-wiki-part1-index",
+    "defaultScoringProfile": "",
+    "fields": [
```
Comment (mattmsft) [inline multiple-search-services/part1.json:6]: please don't include the index definition in this sample

File: multiple-search-services/Program.cs [hunk 5]
```diff
@@ -91,7 +91,7 @@ static async Task RunCommand(bool initialize, string query, int page, int pageSi
 
             // Read settings from appsettings.json
             IConfigurationRoot configuration = new ConfigurationBuilder()
-                .AddJsonFile("appsettings.json", optional: true)
+                .AddJsonFile("appsettings.json", optional: false)
```
Comment (mattmsft) [inline multiple-search-services/Program.cs:94]: why are we changing this?

File: .vscode/extensions.json [hunk 6]
```diff
@@ -0,0 +1,6 @@
+{
+  "recommendations": [
+    "ms-azuretools.vscode-azurefunctions",
```
Comment (mattmsft) [inline .vscode/extensions.json:3]: please remove all these extra .vscode files

File: .vs/VSWorkspaceState.json [hunk 7]
```diff
@@ -0,0 +1,6 @@
+{
+  "ExpandedNodes": [
+    ""
+  ],
+  "PreviewInSolutionExplorer": false
```
Comment (mattmsft) [inline .vs/VSWorkspaceState.json:5]: remove this file

**Review by mattmsft — CHANGES_REQUESTED** [review]

File: multiple-search-services/README.md (hunk 2 above)
Comment (farzad528) [reply multiple-search-services/README.md]: Done, please review.

File: multiple-search-services/appsettings.json (hunk 3 above)
Comment (farzad528) [reply multiple-search-services/appsettings.json]: Done, please review.

File: multiple-search-services/Program.cs (hunk 5 above)
Comment (farzad528) [reply multiple-search-services/Program.cs]: Reverted, please review.

File: .vscode/extensions.json (hunk 6 above)
Comment (farzad528) [reply .vscode/extensions.json]: Done, please review.

**Pull request merged**